content/en/docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -53,6 +53,25 @@ Minikube can be installed locally, and runs a simple, single-node Kubernetes clu
 
 You can get basic information about your cluster with the commands `kubectl cluster-info` and `kubectl get nodes`. However, to get a good idea of what's really going on, you need to deploy an application to your cluster. This is covered in the next section.
 
+#### microk8s
+
+On Linux, *microk8s* might be a good alternative for a local install of upstream Kubernetes. Microk8s keeps up with the current Kubernetes, and auto-updates to the latest point release for your convenience. Microk8s does not require you to install a hypervisor or launch a virtual machine, and installs within ~30 seconds.
```

Comment:
In what cases might it be a good alternative? Also what's the meaning of "upstream Kubernetes" since these are in fact the upstream Kubernetes docs?

"Auto-updates" -> "automatically updates"

"With microk8s, you don't need to install a local..."

Can you wrap your lines at 80 chars? It makes future maintenance easier. I realize the rest of the file doesn't adhere to this now.